doc/manual/plugins.rst

```diff
@@ -46,7 +50,6 @@ Global plugins
    plugins_global/errors
    plugins_global/rc
    plugins_global/wcsmatch
-   plugins_global/changehistory
```

Comment:
Yes, this removes the individual sub-pages. But it also reduces the hierarchical level, so `docutils` will now show slightly bigger font (you asked me back in #451 why the fonts so small).

To retain the sub-pages, the section title needs to be in the RST of the sub-page, not the docstring. Otherwise, Sphinx issues errors. As a result, you won't see the section title (e.g., **ChangeHistory**) on your offline help page generated by `docutils`. And if I leave the section title in both the RST and the docstring, Sphinx will have duplicate sections (and probably error messages too).

Ideas welcome.